site/cypress/e2e/Cypress-UI/pdfs.spec.js

```diff
@@ -20,6 +20,15 @@ function select_gradeable() {
     cy.get('#pageContainer1').should('be.visible');
 }
 
+function pdf_buttons(student=false) {
+    cy.get('[data-testid="save-pdf-btn"]').should('be.visible');
+    cy.get('[data-testid="clear-pdf-btn"]').should('be.visible');
+    if (!student) {
+        cy.get('[data-testid="download-annotations-btn"]').should('be.visible');
+        cy.get('[data-testid="toggle-annotations-btn"]').should('be.visible');
+    }
```

Comment:
Actually, can you add an else statement here to make sure it's not visible to students?